ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/targetdefinition/TargetEditor.java

```diff
@@ -184,6 +186,9 @@ public void doSave(IProgressMonitor monitor) {
 		fDirty = false;
 		editorDirtyStateChanged();
 		fInputHandler.setSaving(false);
+
+		TargetPlatformService service = (TargetPlatformService) TargetPlatformService.getDefault();
```

Comment:
Why is this needed? Should not saving the target file already trigger an event by default?

Reply:
Only if it's the active target.